src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM.java

```diff
@@ -221,6 +221,13 @@ public ChangeLogParser createChangeLogParser() {
         return gitSCM.getBranches();
     }
 
+    public List<UserRemoteConfig> getUserRemoteConfigs() {
```

Comment:
/nit alphabetical order (so add this after `public String getServerId`).

Reply:
Thanks! Done